### PR TITLE
refactor: fix primordials for WebGPU

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: ext/webgpu/01_webgpu.js [hunk 1]
```diff
@@ -715,7 +715,7 @@
       const validationFilteredPromise = PromisePrototypeCatch(
         operation,
         (err) => {
-          if (ObjectPrototypeIsPrototypeOf.prototype(GPUValidationError, err)) {
+          if (ObjectPrototypeIsPrototypeOf(GPUValidationError, err)) {
```
Comment (crowlKats) [inline ext/webgpu/01_webgpu.js]:
```suggestion
          if (ObjectPrototypeIsPrototypeOf(GPUValidationError.prototype, err)) {
```

**Review by crowlKats — APPROVED** [review]
LGTM

**Comment by AaronO** [comment]
@crowlKats @bartlomieju We should upstream this change to https://github.com/gfx-rs/wgpu/

**Comment by crowlKats** [comment]
Yes, i am aware. was going to do that later along the other changes that have been made to our webgpu

**Pull request merged**